core/src/bundle/usBundle.cpp

```diff
@@ -167,6 +167,19 @@ void Bundle::Stop()
     {
       d->bundleActivator->Stop(d->bundleContext);
     }
+	// remove the cached bundle context. TODO : This block can be moved
```

Comment:
A comment as to why the cached bundle context needs to be cleared would be helpful.